src/main/java/trade/wayruha/mexc/enums/TopLimit.java

```diff
@@ -0,0 +1,14 @@
+package trade.wayruha.mexc.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum TopLimit {
```

Comment:
is it Order Book Depth? should we rename it?

Reply:
As you wish. It's represent "Top bids and asks, Valid are 5, 10, or 20."